server/src/utils/ServerState.ts

```diff
@@ -5,27 +5,28 @@ let serverReady = true;
 
 const setReady = (ready: boolean) => {
   LOGGER.log(`Updating server state to ${ready ? 'release' : 'hold'} traffic`);
-
   serverReady = ready;
 };
 
 const isReady = () => serverReady;
 
 const setGlobalTournamentValid = async (prisma: PrismaClient, valid: boolean) => {
-  await prisma.serverState.upsert({
-    where: { id: 1 },
-    update: {
-      globalTournamentValid: valid,
-    },
-    create: {
+  const serverState = await prisma.serverState.findFirst({
+    where: {
       globalTournamentValid: valid,
     },
```

Comment:
This findFirst should not have a where clause, as we simply want to find the first row.